### PR TITLE
Update README.md for clarity and concision

### LINKED ISSUE #7690: The README.md contains typos and could be written more concisely
# READ AND FOLLOW THESE INSTRUCTIONS CAREFULLY

*ISSUES THAT DO NOT CONTAIN NECESSARY INFORMATION MAY BE CLOSED, IMMEDIATELY*

The issue tracker is NOT the place for general support. For questions and
technical assistance, come ask the [Bokeh mailing list](https://groups.google.com/a/continuum.io/forum/#!forum/bokeh) or join the chat on [Gitter](https://gitter.im/bokeh/bokeh). For feature requests, please provide a detailed description or proposal of the new capability or behavior.

For defects or deficiencies, please provide ALL OF THE FOLLOWING:

#### ALL software version info (bokeh, python, notebook, OS, browser, any other relevant packages)
This issue is related to the README.md file contained in the master branch.

#### Description of expected behavior and the observed behavior
The README.md file contains several typos and grammatical errors, and its information could be stated more concisely.

#### Complete, minimal, self-contained example code that reproduces the issue
Open the README.md file included with the master branch.

```
# code goes here between backticks
# N/A
```

#### Stack traceback and/or browser JavaScript console output
N/A

#### Screenshots or screencasts of the bug in action
N/A: Please see the differences in the before/after versions of the README.md edits.

### PR DESCRIPTION
**Pull request by b1skit**
Reworded several sentences and paragraphs for clarity and concision. Corrected several spelling and grammatical errors. All original information was maintained.

All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [x] issues: fixes #7690
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,27 +254,27 @@ streaming datasets.
...
+The easiest way to install Bokeh is using the [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and its included *Conda* package management system. This allows Bokeh and its required dependencies to be installed by entering the following command at a Bash or Windows command prompt:
 
 ```
 conda install bokeh
 ```
 
-This installs Bokeh and all needed dependencies.
-
-To begin using Bokeh or to install using `pip`, follow the [Quickstart](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html) documentation.
+The Bokeh [quickstart documentation](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html) provides information on installing Bokeh using [`pip`](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html#quick-installation). It also includes a [getting started](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html#getting-started) guide.
```
Comment (bryevdv) [inline README.md]: "pip" wil make a very small visual target for users to click. Also for parallelism I think this phrase should begin with another appositive to mirror the "to install" above. Also, the the "getting started" section is useful regardless of the method of installation. Here is seems like it only applies to the pip instructions. Perhaps:

> To install using `pip`, enter the following command at a Bash or Windows command prompt:
> ```
> pip install bokeh
> ```
>
> For more information, refer to the [installation documentation](https://bokeh.pydata.org/en/latest/docs/installation.html).
> 
> Once Bokeh is installed, check out the [Getting Started](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html#getting-started) section of the [Quickstart](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html) .

File: README.md [hunk 2]
```diff
@@ -255,27 +254,27 @@ streaming datasets.
 
 Installation
 ------------
-We recommend using the [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and conda to install Bokeh. Enter this command at a Bash or Windows command prompt:
+The easiest way to install Bokeh is using the [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and its included *Conda* package management system. This allows Bokeh and its required dependencies to be installed by entering the following command at a Bash or Windows command prompt:
```
Comment (bryevdv) [inline README.md]: This seems more wordy and complicated to me. Again I prefer imperative call to action: *To do task X, perform action Y*.

**Comment by b1skit** [comment]
Hi bryevdv, thanks for your feedback. I agree with your points, I'll make some more changes.

**Comment by b1skit** [comment]
@bryevdv Sorry for the spam, a bit of a n00b question here: I still see "1 review requesting changes." I'm pretty sure I addressed all of your comments. Is there something I missed or that need to click to pass this back to you?

Thanks,

**Comment by bryevdv** [comment]
No it's just up to a maintainer to review again, I will look at things tommorow, it's late :)

**Comment by bryevdv** [comment]
@b1skit I've approved the changes. There was a spurious test failure, so I have restarted the CI  build and will merge later today when everything is green. Thanks for the PR!

**Pull request merged**